test/lib/services/time_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+require 'wovnrb/services/time'
+
+module Wovnrb
+  class TimeTest < WovnMiniTest
+    def test_round_down_time
+      assert_equal(0, TimeUtil.round_down_time(0, 10))
+      assert_equal(10, TimeUtil.round_down_time(10, 10))
+      assert_equal(10, TimeUtil.round_down_time(16, 10))
+      assert_equal(30, TimeUtil.round_down_time(30, 15))
+      assert_equal(100, TimeUtil.round_down_time(100, 10))
+    end
+
+    def test_time_proc
+      current_time = Time.now.utc.sec
+      assert((TimeUtil.time_proc.call - current_time) < 2.0)
```

Comment:
this test doesn't look useful. There's not any logic here, it is just a wrapper for `Time.now` so i don't think you need to test it.

Reply:
Yes this test might seem very redundant. But this is the only check that makes sure the `time_proc` is usable (we mocked it in all other tests). It's basically to make sure 1) the proc is actually callable and it is a proc and 2) it's not returning something totally unexpected (like a datetime object for example)

Reply:
Timecop would still be useful if you wanted to write this more strictly 😉

Reply:
I removed time_proc and tested with Timecop.